File: TrackMyAct.Server/Models/Entities/CommentEntity.cs
public class CommentEntity : Entity
{
    public string Text { get; set; }
    public CommentType Type { get; set; }

    public int Mark { get; set; }

    public ParticipantEntity Author {get; set; }
    public int AuthorId { get; set; }

    public EventEntity Event { get; set; }
    public int? EventId { get; set; }

    public OrganizerEntity Organizer { get; set; }
    public int? OrganizerId { get; set; }
}

File: TrackMyAct.Mobile/composeApp/src/commonMain/kotlin/org/js/tma/dto/CommentEntity.kt
package org.js.tma.dto

import kotlinx.serialization.Serializable

@Serializable
data class CommentEntity(
    override var id: Int,
    var text: String,
    var type: CommentType,
    var mark: Int,

    var author: ParticipantEntity?,
    var authorId: Int,

    var event: EventEntity?,
    var eventId: Int?,

    var organizer: OrganizerEntity?,
    var organizerId: Int?
) : Entity()